airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -244,6 +308,11 @@ def read(self) -> Iterable[Record]:
     def to_slice(self) -> Optional[Mapping[str, Any]]:
         return self._slice
 
+    def identifier(self) -> Optional[Mapping[str, Any]]:
```

Comment:
should we remove the property entirely until it is needed?

Reply:
Good point! I removed it from the interface but not the implementation. Thanks for pointing that out!